processor/processor.go

```diff
@@ -230,20 +230,22 @@ func setupCountAs() {
 						printDebug(fmt.Sprintf("set to count extension: %s as language %s by language", t[0], name))
 					}
 				}
+
+				if !identified {
+
+				}
```

Comment:
This looks like you committed  something by accident.

Reply:
How did I manage that... oh well.